app/views/dashboard/_polling.html.erb

```diff
@@ -1,3 +1,5 @@
 <script>
-  MinionPoller.poll();
+  document.addEventListener("DOMContentLoaded", function(event) { 
```

Comment:
jQuery is available in this project, I'd suggest to use it instead.
Take a look at https://api.jquery.com/ready/